plugins/search-backend-module-catalog/src/module.ts

```diff
@@ -60,7 +59,7 @@ export const catalogCollatorExtensionPoint =
  *
  * @public
  */
-export default createBackendModule({
+export const searchBackendModule = createBackendModule({
```

Comment:
any reason to not follow the naming patterns here? https://backstage.io/docs/backend-system/architecture/naming-patterns. I'd guess this would end up being `searchModuleCatalogCollator` or something like that?

Reply:
Good point, renamed! It isn't exported with that name, but indeed this was just me forgetting about the pattern.